### PR TITLE
Update Ruby Gem version to 1.9.3

### PR DESCRIPTION
**Pull request by tirodkar**

### CODE REVIEW

**Comment by khandpur** [comment]
Weren't we going to move this to 1.9.3 ?

**Comment by tirodkar** [comment]
Will do.

**Comment by tirodkar** [comment]
Done!!!

File: docs/versions.md [hunk 1]
```diff
@@ -25,4 +25,4 @@ EarlGrey Release Version                                       | Corresponding E
 [1.9.0](https://github.com/google/EarlGrey/releases/tag/1.9.0) | [0.2.0](https://rubygems.org/gems/earlgrey/versions/0.2.0) | 1.0.0
 [1.9.1](https://github.com/google/EarlGrey/releases/tag/1.9.1) | [0.2.0](https://rubygems.org/gems/earlgrey/versions/0.2.0) | 1.0.0
 [1.9.2](https://github.com/google/EarlGrey/releases/tag/1.9.2) | [0.2.1](https://rubygems.org/gems/earlgrey/versions/0.2.1) | 1.0.0
-[1.9.3](https://github.com/google/EarlGrey/releases/tag/1.9.3) | [0.2.2](https://rubygems.org/gems/earlgrey/versions/0.2.2) | 1.0.0
+[1.9.3](https://github.com/google/EarlGrey/releases/tag/1.9.3) | [0.2.2](https://rubygems.org/gems/earlgrey/versions/1.9.3) | 1.0.0
```
Comment (sid-github) [inline docs/versions.md]: [0.2.2] => [1.9.3]

Comment (tirodkar) [inline docs/versions.md]: Done! Thanks.

**Pull request merged**